kong/plugins/datadog/handler.lua

```diff
@@ -86,7 +87,10 @@ local function send_entries_to_datadog(conf, messages)
                                 message.service and gsub(message.service.name ~= null and
                                 message.service.name or message.service.host, "%.", "_") or "",
                                 message.response and message.response.status or "-",
-                                consumer_id, metric_config.tags, conf)
+                                consumer_id, metric_config.tags,
+                                conf,
+                                message.route and (message.route.name and gsub(message.route.name ~= null and
```

Comment:
is this passing `false` to gsub when `message.route.name` is `null`?
When it gets this complex I'd rather be verbose and write down the logic above, assigning the final value to some local variable, and then just pass it to `compose_tags`, wdyt?